### ISSUE TITLE
Vercel Deployment failed with the following error:

### ISSUE THREAD

**Issue by jsteinich** [hashicorp/terraform-cdk#1637]
Deployment failed with the following error:
~~~
The provided GitHub repository does not contain the requested branch or commit reference. Please ensure the repository is not empty.
~~~

_Originally posted by @vercel[bot] in https://github.com/hashicorp/terraform-cdk/issues/1586#issuecomment-1074236962_

**Comment by ansgarm** [comment]
Thanks for raising this, I mentioned it here: https://github.com/hashicorp/terraform-cdk/pull/1608

**Comment by ansgarm** [comment]
This was resolved via #1608
